host-bmc/host_pdr_handler.cpp

```diff
@@ -149,7 +149,7 @@ HostPDRHandler::HostPDRHandler(
                 this->stateSensorPDRs.clear();
                 fruRecordSetPDRs.clear();
                 isHostOff = true;
-                isHostTransitioningToOff = false;
+                isHostRunning = false;
```

Comment:
isHostRunning should also be reset when host goes to any state apart from Running.

Reply:
yes, made the changes.